Tests/lib/vsts-task-lib/task.ts

```diff
@@ -608,9 +608,19 @@ export class TestPublisher {
             properties['config'] = config;
         }
 
-        for (var i = 0; i < resultFiles.length; i++) {
-            command('results.publish', properties, resultFiles[i]);
+        if (runTitle) {
+            properties['runTitle'] = runTitle;
```

Comment:
In case no run title is provided, is the behavior same as before (for run titles)?